src/main/java/controller/CreateChat.java

```diff
@@ -45,6 +46,56 @@
     @Autowired
     private UniqueIdGenerator uniqueIdGenerator;
 
+    /**
+     * Encapsulation for the following fields related to a Chat for one of the Users engaged in that Chat. 
+     * <ol>
+     * <li> ChatId </li>
+     * <li> UserID of Second User in Chat </li>
+     * </ol>
+     */
+    public static final class ChatIdWithUserIds {
```

Comment:
Does this correspond to a new table?
If yes, please update the schema file as well.

Reply:
No, no, just a `class` for receiving `Spanner` query results for `createChat`